### PR TITLE
Adding fcsmakers norms

### PR DESCRIPTION
**Pull request by Mr-Darfler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 
 ## Norms
-I **learn** best when: 
+I **learn** best when: There are very few distractions
 
-So I want a classroom that is:
+So I want a classroom that is: Free of interruptions
 
-In order to acheive this, I proposet the following classroom norm:
+In order to acheive this, I propose the following classroom norm: Take care of your personal needs before coming to class on time.
```
Comment (fcsmakers) [inline README.md]: @fcsmakers I think that this is actually two different norms, one about taking care of your personal needs and one about arriving on time.

Comment (fcsmakers) [inline README.md]: I think that this is actually two different norms. One about taking care of your personal needs and one about arriving on time. @Mr-Darfler

File: README.md [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
 
 
-I **contribute** best when:
+I **contribute** best when: When I am asked directly for feedback
 
-So I want a classroom that is:
+So I want a classroom that: Asks good questions
 
-In order to acheive this, I proposet the following classroom norm:
+In order to acheive this, I propose the following classroom norm: Be direct and constructive when providing feedback.
```
Comment (fcsmakers) [inline README.md:18]: Great Norm.

File: README.md (hunk 1 above)
Comment (Mr-Darfler) [reply README.md]: Thanks. I will turn this into two different norms

**Comment by Mr-Darfler** [comment]
Great Conversation. I'm will merge this branch

**Pull request merged**